build_projects/dotnet-cli-build/dotnet-cli-build.csproj

```diff
@@ -45,11 +45,14 @@
       <Version>7.2.1</Version>
     </PackageReference>
     <PackageReference Include="NuGet.CommandLine.XPlat">
-      <Version>4.0.0-rc2</Version>
+      <Version>4.0.0-rc3</Version>
     </PackageReference>
     <PackageReference Include="Microsoft.Build.Utilities.Core">
       <Version>$(CLI_MSBuild_Version)</Version>
     </PackageReference>
+    <PackageReference Include="Microsoft.Build.Framework">
```

Comment:
because restore was failing due to the fact that nuget.commandline.xplat was bringing in an older version of microsoft.build.framework than $(CLI_MSBUILD_VERSION) . it could most likely be a restore bug, but @emgarten recommended to get around it by adding the dependency on the top level so it always wins.